SU-HSDO/suhumsci
# STN-595  Hotfix: for colorful category pipes

# READY FOR REVIEW

## Summary
The category pipe was no longer appearing on blocks that were not listing the categories in a `<ul>`. Blocks using categories in divs, did not have the pipes at all.

This fix gives width to the category pipe while also retaining the transform property that helps address the category pipe sizing from being inconsistent as it was in the initial bug.

Jira Card: [STN-595](https://sparkbox.atlassian.net/browse/STN-595)
Previous bug card: [STN-554](https://sparkbox.atlassian.net/browse/STN-554)

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. npm run test and make sure all tests pass
2.Go to http://sparkbox-sandbox.suhumsci.loc/
3. Set the theme to Colorful
4. Go to http://sparkbox-sandbox.suhumsci.loc/people/ran-abramitzky
5. The category pipe on the block on the bottom right should be visible and consistent.
6. Zoom your browser window to 80%. (This was the only way we could replicate the issue from STN-554.) Make sure the pipes stay consistent and do not change sizes.
4. Please also confirm these pipes are still accurate on the home page event cards as well as the categories listed on the People page: http://sparkbox-sandbox.suhumsci.loc/people-0. (Zoom browser to confirm consistent sizing)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
